test/extended/olm/olmv1.go

```diff
@@ -163,3 +159,14 @@ var _ = g.Describe("[sig-olmv1] OLMv1 operator installation", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
 })
+
+func checkTestSkip(ctx context.Context, oc *exutil.CLI) {
+	if !exutil.IsTechPreviewNoUpgrade(ctx, oc.AdminConfigClient()) {
```

Comment:
is there a way for tests to check the [`NewOLM` feature gate][1], so you have one less thing you need to bump when that is added to the default feature set?  This line hard-codes the current state, where [the `NewOLM` feature gate is in tech-preview, but not yet in the default feature set][2].

[1]: https://github.com/openshift/api/pull/2061/files#diff-fdab2a007eec159636f78603e2daf39204052afecb564d7ccbf715f881106c44R18
[2]: https://github.com/openshift/api/blob/a2817b89f7e0989016967055a142d5e88bed18de/features/features.go#L493-L498

Reply:
Done, I think...

Reply:
I didn't get rid of the TechPreview check yet, because I wasn't sure of the status of the FeatureGate.